localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
Sorry, I didn't see your response before I added the note about the wikipedia name which is why it seems like I didn't understand your response. 

The field name should primarily make sense to Bulgarian developers using it. It's also nice to have the `BG` prefex for consistency with other localflavors. The prefix also allows non-Bulgarian developers to quickly see that it's a field for Bulgaria. With these goals in mind, you are in the best position to choose between the English translation of the Bulgarian names or the acronyms (`BGEGNField`, `BGEIKField`). It seems to me that the acronyms are a better choice.

Thanks for the explanation.